### PR TITLE
Update doc for deployment to add Cloudstack support

### PR DESCRIPTION
**Pull request by AliceAlbano**

### CODE REVIEW

File: end-user/en/source/pages/templating/deploy-machine-image.rst [hunk 1]
```diff
@@ -45,6 +45,10 @@ For Amazon, only full accounts can be used, trial accounts are not supported for
 
 	.. note:: If you want to stop your deployment, you can do so by clicking on the delete (garbage) icon. This will stop it and remove the instance from your cloud.
 
+	.. note:: Deployment on Cloudstack platform does not handle the possibility to give a floating IP to a virtual machine.
```
Comment (emuus) [inline end-user/en/source/pages/templating/deploy-machine-image.rst]: replace "does not handle the possibility to give" by "does not allow a floating IP"

File: end-user/en/source/pages/templating/deploy-machine-image.rst [hunk 2]
```diff
@@ -45,6 +45,10 @@ For Amazon, only full accounts can be used, trial accounts are not supported for
...
+
+	.. warning:: All deployed virtual machines using this procedure will have ssh port (22) open in their security group.
```
Comment (emuus) [inline end-user/en/source/pages/templating/deploy-machine-image.rst]: put this warning before the first note (line 45) as it is more important than the note.

**Comment by AliceAlbano** [comment]
Thanks @emuus for your review, I have made the changes you asked for.

File: end-user/en/source/pages/templating/deploy-machine-image.rst [hunk 3]
```diff
@@ -43,8 +43,12 @@ For Amazon, only full accounts can be used, trial accounts are not supported for
 
 		.. image:: /images/machine-image-deploy-status.png
 
+	.. warning:: All deployed virtual machines using this procedure will have ssh port (22) open in their security group.
```
Comment (jbremond) [inline end-user/en/source/pages/templating/deploy-machine-image.rst]: This more info and not warning

File: end-user/en/source/pages/templating/deploy-machine-image.rst [hunk 4]
```diff
@@ -43,8 +43,12 @@ For Amazon, only full accounts can be used, trial accounts are not supported for
 
 		.. image:: /images/machine-image-deploy-status.png
 
+	.. note:: All deployed virtual machines using this procedure will have ssh port (22) open in their security group.
+
 	.. note:: If you want to stop your deployment, you can do so by clicking on the delete (garbage) icon. This will stop it and remove the instance from your cloud.
 
+	.. note:: Deployment on Cloudstack platform does not allow a floating IP to a virtual machine.
```
Comment (jbremond) [inline end-user/en/source/pages/templating/deploy-machine-image.rst]: I think we can remove this line now

Comment (AliceAlbano) [reply end-user/en/source/pages/templating/deploy-machine-image.rst]: Thanks for the review, I have removed the line.

**Pull request merged**